packages/tracer/src/Tracer.ts

```diff
@@ -415,20 +415,20 @@ class Tracer extends Utility implements TracerInterface {
       // The descriptor.value is the method this decorator decorates, it cannot be undefined.
       // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
       const originalMethod = descriptor.value!;
-      
-      descriptor.value = (...args: unknown[]) => {
+
+      descriptor.value = ((...args: unknown[]) => {
         if (!this.isTracingEnabled()) {
-          return originalMethod.apply(target, [...args]);
+          return originalMethod.apply(target, [ ...args ]);
```

Comment:
This is mostly a styling comment, but could you please revert all the changes to this file except the one that contains the fix? (here I'm referring to the tabs/spaces, extra parenthesis, extra spaces, extra lines.

It's a bit difficult to see the actual diff & these would not be in line with the rest of the codebase.

Reply:
Sorry did not see your comment at first, i have fixed this and re-committed